db/institutions.yml

```diff
@@ -262,4 +262,108 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+vanderbilt:
+  country: USA
+  aka:
+  - Vanderbilt
+  - vanderbilt
+  city: Nashville
```

Comment:
does that sheet have the street address?  I have been adding ``street`` fields

Reply:
Can I get that in another round? We will need to do more advanced munging to get the correct street name.